ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -301,9 +305,13 @@ export default defineComponent({
       categoryMessage: computed((): string => {
         return localState.validations?.category?.message || ''
       }),
+      isPPROnly: computed((): boolean => hasPprRole.value && !(hasMhrRole.value && getFeatureFlag('mhr-ui-enabled'))),
```

Comment:
I think* the && condition is the same as the new `hasMhrRoleEnabled` property, might be useful here as well.

Reply:
Right, I meant to update it, thanks

Reply:
Updated